packages/internal-test-env/index.ts

```diff
@@ -87,36 +87,33 @@ export interface EnvVariables {
 }
 
 export function setupEnv() {
```

Comment:
I'd rather keep `setupEnv` as simple as possible as it's called in a lot of places.. we've always had it return early in CI (this dates back before we had this library)

Reply:
Can the bulk of the changes to this method be reverted? Unless of course you'd argue that they give more clarity to the method?

Reply:
https://github.com/inrupt/typescript-sdk-tools/pull/113/commits/9f8f10f1b8cf1c0c52b333146501a3521c1406f8